x-pack/test/accessibility/apps/index_lifecycle_management.ts

```diff
@@ -55,6 +58,31 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await esClient.ilm.deleteLifecycle({ policy: TEST_POLICY_NAME });
     });
 
+    it('Create policy Wizard', async () => {
+      await retry.waitFor('Index Lifecycle Policy create/edit view to be present', async () => {
+        return testSubjects.isDisplayed('createPolicyButton');
+      });
+
+      // Navigate to create policy page and take snapshot
+      await testSubjects.click('createPolicyButton');
+      await retry.waitFor('Index Lifecycle Policy create/edit view to be present', async () => {
+        return (await testSubjects.getVisibleText('policyTitle')) === 'Create policy';
+      });
+      await a11y.testAppSnapshot();
+
+      // Fill out form after enabling all phases and take snapshot.
+      await indexLifecycleManagement.fillNewPolicyForm('testPolicy', true, true, true);
+      await a11y.testAppSnapshot();
```

Comment:
I'd break out each `await a11y.testAppSnapshot()` into its own test. 

It makes it a lot easier when debugging because the stack traces are often not very helpful. It often just shows a test failure somewhere inside of axe which isn't helpful to us so we don't actually know which snapshot failed.

But if each snapshot is it's own test, then we know exactly what test failed without having to rely on the stack trace.

Reply:
Got it!